muckrock/communication/models.py

```diff
@@ -189,7 +189,7 @@ class PhoneNumber(models.Model):
     objects = PhoneNumberQuerySet.as_manager()
 
     def __str__(self):
-        number = f"{self.number.as_national} ({self.type})"
```

Comment:
Having the fax/phone distinction is useful on the backend, maybe make another method if you want a way to just display the number?

Reply:
Fixed this